lua/zenbones/specs/dark.lua

```diff
@@ -253,6 +253,17 @@ local function generate(p, opt)
 			TSWarning               { WarningMsg },
 			TSDanger                { Error },
 
+
```

Comment:
Single space so it will look consistent.

Reply:
👍